wazuh/wazuh-kibana-app
# Fix statistics visualizations when a node is selected

Hi team this PR solves:
- The graph for `Events processed` and `Events dropped` is showing data of all nodes.

Closes #3068 

### How to Test:
To test this PR you need a worker node, then you should go to Management > Statistics > Analysis Engine. There you have to select a node and the graphs must show only data about this node.

![Captura de pantalla 2021-03-29 a las 13 50 06](https://user-images.githubusercontent.com/46006326/112832511-b5d39000-9095-11eb-8e92-c5b39b37918f.png)
